javascript/javascript1/week2/event-application.js

```diff
@@ -0,0 +1,35 @@
+getEventWeekday = (daysToEvent) => {
+  const d = new Date();
+  const today = d.getDay();
+
+  const q = (daysToEvent + today) % 7;
+
+  for (i = 1; i <= 7; i++) {
+    switch (q) {
+      case 1:
+        return "Monday";
+        break;
+      case 2:
+        return "Tuesday";
+        break;
+      case 3:
+        return "Wednsday";
+        break;
+      case 4:
+        return "Thursday";
+        break;
+      case 5:
+        return "Friday";
+      case 6:
+        return "Saturday";
+      case 7:
+        return "Saunday";
+
+        break;
+      default:
+        return "Error XD!";
+    }
+  }
+};
+
+console.log("The event is being held on " + getEventWeekday(9));
```

Comment:
great job with this switch statement but it'll be much simpler if you used an array with the days of the week and put the logic in a function , can you do that?

Reply:
just did it and committed